packages/account/Readme.md

```diff
@@ -0,0 +1,133 @@
+
+**paymasterUrl** you can get this value from biconomy dashboard. https://dashboard.biconomy.io
+
+```typescript
+// This is how you create paymaster instance in your dapp's
+import { IPaymaster, BiconomyPaymaster } from '@biconomy/paymaster'
+
+  const paymaster: IPaymaster = new BiconomyPaymaster({
+    paymasterUrl: '' // you can get this value from biconomy dashboard. https://dashboard.biconomy.io
+  })
+```
+
+## Smart Account instance configuration
+
+| Key           | Description |
+| ------------- | ------------- |
+| signer        | This signer will be used for signing userOps for any transactions you build. You can supply your your EOA wallet signer|
+| chainId       | This represents the network your smart wallet transactions will be conducted on. Take a look following Link for supported chain id's |
+| rpcUrl        | This represents the EVM node RPC URL you'll interact with, adjustable according to your needs. We recommend to use some private node url for efficient userOp building|
+| paymaster     | you can pass same paymaster instance that you have build in previous step. Alternatively, you can skip this if you are not interested in sponsoring transaction using paymaster|
+|               | Note: if you don't pass the paymaster instance, your smart account will need funds to pay for transaction fees.|
+| bundler       | You can pass same bundler instance that you have build in previous step. Alternatively, you can skip this if you are only interested in building userOP|
+
+
+## Example Script Usage
+
+```typescript
+// This is how you create BiconomySmartAccount instance in your dapp's
+
+import { BiconomySmartAccount, BiconomySmartAccountConfig } from "@biconomy/account"
+import { IHybridPaymaster, PaymasterMode, SponsorUserOperationDto, PaymasterFeeQuote } from '@biconomy/paymaster'
+
+const biconomySmartAccountConfig: BiconomySmartAccountConfig = {
+    signer: wallet.getSigner(),
+    chainId: ChainId.POLYGON_MAINNET, 
+    rpcUrl: '',
+    paymaster: paymaster, //you can skip paymaster instance if not interested in transaction sposoring
+    bundler: bundler,
+}
+
+const biconomyAccount = new BiconomySmartAccount(biconomySmartAccountConfig)
+const biconomySmartAccount =  await biconomyAccount.init()
+
+// native token transfer transaction creation
+// you can create any sort of transaction following same structure
+const transaction = {
+  to: '0x85B51B068bF0fefFEFD817882a14f6F5BDF7fF2E',
+  data: '0x',
+  value: ethers.utils.parseEther('0.1'),
+}
+
+// building userOperation
+const builtUserOp = await biconomySmartAccount.buildUserOp([transaction])
+
+const biconomyPaymaster = biconomySmartAccount.paymaster as IHybridPaymaster<SponsorUserOperationDto>
```

Comment:
this part is specific to paymaster and biconomy paymaster. this should not be here but instead in paymaster readme.